docs/examples/rails-api/configmap.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-map
+  labels:
+    application: web
```

Comment:
better to be `app.kubernetes.io/name: rails-api`, list of recommended labels: https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/

Reply:
I've had issues with lookups on name versus app.kubernetes.io/name.  It's probably my own understanding and how to use them.  I do need to fix the labels on this config.  Will add a PR.